extensions/amp-iframe/0.1/amp-iframe.js

```diff
@@ -466,6 +469,13 @@ export class AmpIframe extends AMP.BaseElement {
       /*opt_allowOpaqueOrigin*/ true
     );
 
+    // Listen for resize messages sent by Pym.js.
+    this.unlisten_ = listen(
+      window,
```

Comment:
s/window/this.win

Reply:
Done in 89dcadb412a47b1620553987ecd11b02b03c66d9